### PR TITLE
Upgrade packages minors and fix deep selector support 

### PR DESCRIPTION
**Pull request by skjnldsv**
Upgrading
- @babel/core from `^7.16.7` to `^7.19.6`
- babel-loader from `^8.2.3` to `^8.2.5`
- css-loader from `^6.5.1` to `^6.7.1`
- sass from `^1.47.0` to `^1.55.0`
- sass-loader from `^13.0.1` to `^13.1.0`
- vue-loader from `^15.9.8` to `^15.10.0`
- vue-template-compiler from `^2.7.0` to `^2.7.13`
- webpack from `^5.66.0` to `^5.74.0`
- webpack-cli from `^4.9.1` to `^4.10.0`
- webpack-dev-server from `^4.7.2` to `^4.11.1`

### CODE REVIEW

File: tests/src/views/MainView.vue [hunk 1]
```diff
@@ -44,5 +44,9 @@ export default {
 <style lang="scss" scoped>
 html {
 	background-color: red;
+
+	:deep(.test) {
+		background-color: blue;
+	}
```
Comment (artonge) [inline tests/src/views/MainView.vue:50]: Why is this added ?

Comment (skjnldsv) [reply tests/src/views/MainView.vue:50]: Testing, stylelint

Comment (skjnldsv) [reply tests/src/views/MainView.vue:50]: vue-loader bump includes a deep selector change. :)

**Review by artonge — APPROVED** [review]

**Review by raimund-schluessler — APPROVED** [review]
